ci-operator/step-registry/rosa/aws/sts/hypershift/mp/rosa-aws-sts-hypershift-mp-workflow.yaml

```diff
@@ -0,0 +1,21 @@
+workflow:
+  as: rosa-aws-sts-hypershift-mp
+  steps:
+    env:
+      HOSTED_CP: "true"
+      ZONES_COUNT: "3"
+      COMPUTE_MACHINE_TYPE: "m5.2xlarge"
+      REPLICAS: "3"
+      DISABLE_WORKLOAD_MONITORING: "false"
+      MP_MACHINE_TYPE: "r5.xlarge"
+      MP_REPLICAS: "2"
```

Comment:
Just curious can I create more than 1 MP. Not an immediate requirement for perfscale.
for example if I need a MP for `infra` nodes and one more for  `workload` nodes with different labels and taints.
I was thinking of a dictionary to take inputs 
```yaml
EXTRA_MACHINEPOOL:
- MP_NAME: Infra
  MP_REPLICAS: 1
- MP_NAME: Workload
  MP_REPLICAS:1
```
I tried to but prow validator didnt allow me to do so, `Go struct field MultiStageTestConfiguration.tests.steps.env of type string`

Reply:
You can call `rosa-conf-machine-pool` two times to create machine pool with the same configuration. And then in your step, you can choose one to be infra, and another one to worker.

Reply:
right, we can work it around.